modules/registry-exposing-secure-registry-manually.adoc

```diff
@@ -9,7 +9,7 @@
 Instead of logging in to the {product-title} registry from within the cluster,
 you can gain external access to it by exposing it with a route. This allows you
 to log in to the registry from outside the cluster using the route address, and
-to tag and push images using the route host.
+to tag and push images using the route host of your project.
```

Comment:
"to tag and push images to an existing project using the route host."

Reply:
Thanks @ricardomaraschini . I have made the suggested changes.